src/linux/InotifyEventLoop.cpp

```diff
@@ -137,6 +137,10 @@ void InotifyEventLoop::work() {
         inotifyService->removeDirectory(event->wd);
       }
     } while((position += sizeof(struct inotify_event) + event->len) < bytesRead);
+    if (renameEvent.isStarted) {
```

Comment:
This is your moved out of watch tree fix, right?

Reply:
Yes exactly, those 4 lines :)

Reply:
Thanks :smile: